releasenotes/notes/kubelet-image-tags-b74a4dc514a33a58.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    The kubelet will now collect image tags from the pod list
```

Comment:
What about "The agent will now collect container image tags when running in kubernetes-only mode" ?